CONTRIBUTING.md

```diff
@@ -71,18 +86,6 @@ To run unit tests, navigate to the root of the repository, then call:
 
 We recommend increasing your GitHub API rate limit before running tests (see above).
 
-## Building JS and CSS
```

Comment:
Do we not need this any more? Maybe we should leave a pointer here for how to rebuild just the JS and CSS for people exclusively working on the frontend.